imgoing-frontend/src/screens/MainScreen.tsx

```diff
@@ -1,25 +1,20 @@
-import React, { useEffect } from 'react';
-import { Text } from 'react-native';
+import React from 'react';
 import styled from 'styled-components/native';
-import CaptionTypo from '../components/typography/CaptionTypo';
-import TitleTypo from '../components/typography/TitleTypo';
+import HomeLanding from '../components/HomeLanding';
 
 const MainScreen = () => {
   return (
     <Wrapper>
-      <TitleTypo lang="en">Main</TitleTypo>
-      <CaptionTypo lang="ko">이건한글</CaptionTypo>
+      <HomeLanding />
```

Comment:
```suggestion
      {data ? <Home /> : <HomeLanding />}
```

데이터가 있을 경우를 명시적으로 일단 넣어 놓는 게 좋을 것 같습니다!

Reply:
이부분은 데이터 넣어서 테스트 해보면서 추후에 수정하겠습니다! 

{data ? <HomeMain/> : <HomaLanding/>}